src/components/atoms/toggle-expand/ToggleExpand.ts

```diff
@@ -37,4 +37,4 @@ export const ToggleExpand = defineComponent({
   },
 });
 
-export const lazy = supportLazy(ToggleExpand);
+export const lazy = supportLazy(ToggleExpand as any);
```

Comment:
@ThaNarie I think this needs a change in MubanJS to get this working without `as any`

Reply:
Solved in newer @muban/muban versions.